twcmanager.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Tesla Wallcharger Manager
+After=syslog.target network.target
+
+[Service]
+Type=simple
+User=pi
+WorkingDirectory=/home/pi/TWCManager
+ExecStart=/usr/bin/python3 -u -m TWCManager
+Restart=on-abort
+
```

Comment:
Could add `Environment=PYTHONUNBUFFERED=1` so that `journalctl -f` follows the log in real time.  Otherwise it has to wait until Python flushes its buffers.

Reply:
no need to set Environment variable, as it is started with `-u`.
tested and verified, all visible in `journalctl -f` as is (with just `-u`)

```
-u     : force the stdout and stderr streams to be unbuffered;
         this option has no effect on stdin; also PYTHONUNBUFFERED=x
```

Reply:
Perfect :+1: Sorry I completely failed to see the `-u` in there o\